wandb/sdk/internal/sender.py

```diff
@@ -1554,27 +1554,32 @@ def get_local_info(self) -> "LocalInfo":
         return local_info
 
     def _flush_job(self) -> None:
+        if (
+            self._job_builder.disable
```

Comment:
what is the meaning of the disable attribute? does it make sense to have the `disable_job_creation` as part of it's definition?

Reply:
Valid point